examples/slack-security-findings/daily_findings.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import annotations
```

Comment:
Why did you want to enable this? AFAIK the only thing this does is move annotations to runtime instead of """compile"""-time. Unlike the other __future__ stuff, there is no Python version where the behavior is enabled by default.

Reply:
I recall I needed it for `TypedDict`, which I'm no longer using.